### PR TITLE
[ci skip] Update Documentation: Add a code example and clarify docs

### PR DESCRIPTION
**Pull request by benkoshy**
### Why this PR?
- To make it easier for developers: where exactly do we apply the configuration settings: `config.action_controller.include_all_helpers = false`? Unless you're a rails expert you won't know: this PR seeks to clarify that. Hopefully it will save people from googling all the answers.

I hope this helps.

### CODE REVIEW

File: actionpack/lib/action_controller/metal/helpers.rb [hunk 1]
```diff
@@ -12,6 +12,12 @@ module ActionController
   # In previous versions of \Rails the controller will include a helper which
   # matches the name of the controller, e.g., <tt>MyController</tt> will automatically
   # include <tt>MyHelper</tt>. To return old behavior set +config.action_controller.include_all_helpers+ to +false+.
+  # You can include the aforementioned in any environment file you wish (e.g. config/environments/production.rb) or alternatively
```
Comment (eugeneius) [inline actionpack/lib/action_controller/metal/helpers.rb]: Changing this behaviour for a specific environment doesn't sound sensible to me - unless you can think of a reason that would be useful, I think we should just recommend setting it in `config/application.rb`.

File: actionpack/lib/action_controller/metal/helpers.rb [hunk 2]
```diff
@@ -12,6 +12,12 @@ module ActionController
   # In previous versions of \Rails the controller will include a helper which
   # matches the name of the controller, e.g., <tt>MyController</tt> will automatically
   # include <tt>MyHelper</tt>. To return old behavior set +config.action_controller.include_all_helpers+ to +false+.
```
Comment (eugeneius) [inline actionpack/lib/action_controller/metal/helpers.rb]: "To return old behavior" is ungrammatical, can you fix that while you're editing this paragraph?

File: actionpack/lib/action_controller/metal/helpers.rb [hunk 3]
```diff
@@ -12,6 +12,12 @@ module ActionController
...
+  # You can include the aforementioned in any environment file you wish (e.g. config/environments/production.rb) or alternatively
+  # you can include it in your config/application.rb file (for example):
```
Comment (eugeneius) [inline actionpack/lib/action_controller/metal/helpers.rb]: I don't think we need "(for example)" here; it's clear that the following code snippet is an example.

File: actionpack/lib/action_controller/metal/helpers.rb (hunk 2 above)
Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: will do!

File: actionpack/lib/action_controller/metal/helpers.rb (hunk 3 above)
Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: ok will remove. 

Shall I squash the commits and resubmit?

Comment (eugeneius) [reply actionpack/lib/action_controller/metal/helpers.rb]: Yes please 👍

File: actionpack/lib/action_controller/metal/helpers.rb (hunk 1 above)
Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: ok.

**Comment by BKSpurgeon** [comment]
@eugeneius Not sure why the build is failing: all i did was change the docs?

**Comment by eugeneius** [comment]
That check is also failing on master right now: https://github.com/rails/rails/runs/459070090#step:4:54

Feel free to ignore it, since it's unrelated to your changes.

File: actionpack/lib/action_controller/metal/helpers.rb [hunk 4]
```diff
@@ -49,7 +54,11 @@ module ActionController
   #   N/A | Carolina Railhawks Training Workshop
   #
   module Helpers
-    extend ActiveSupport::Concern
+    extend ActiveSupport::      # Extract helper names from files in <tt>app/helpers/**/*_helper.rb</tt>
+      def all_application_helpers
+        all_helpers_from_path(helpers_path)
+      end
+    endConcern
```
Comment (eugeneius) [inline actionpack/lib/action_controller/metal/helpers.rb]: This looks accidental, could you undo this change?

Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: Yes, this one is a strange one: without the addition of those spaces, (2) spaces, the build fails: due to rubocop. 

I can revert the change, but the build will not pass. Please confirm.

Comment (eugeneius) [reply actionpack/lib/action_controller/metal/helpers.rb]: There are several new lines added here, and they're not valid Ruby syntax. It looks like a copy and paste error.

Don't worry about placating RuboCop if its complaints are unrelated to your changes.

Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: Ok I see what you mean. That's very strange. I have no idea how they came into this commit. I'll revert it.

Comment (BKSpurgeon) [reply actionpack/lib/action_controller/metal/helpers.rb]: ok i think we have a winner here! thx for the reviewing, i surely appreciate it.

**Comment by eugeneius** [comment]
Thanks!
